### ISSUE TITLE
Edit Snippet?

### ISSUE THREAD

**Issue by poetimp** [faustovaz/gitlab-snippets-plugin#1]
Am I missing something? I can create, view and delete snippets. Can I not edit as well?

**Comment by faustovaz** [comment]
No, you're right. Right now you can't edit snippets, but i'm planning to add this feature soon.

**Comment by poetimp** [comment]
It would be a great add! Thank you!
